Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -482,8 +482,14 @@ private function _getPaymentMethodConfig()
 
         if (!is_null($this->_oTransaction)) {
             $sPaymentId = $this->_oTransaction->getPaymentType();
-            $oPaymentMethod = PaymentMethodFactory::create($sPaymentId);
-            $oConfig = $oPaymentMethod->getConfig();
+
+            if ($sPaymentId) {
+                $oPaymentMethod = PaymentMethodFactory::create($sPaymentId);
+                $oConfig = $oPaymentMethod->getConfig();
+            } else { // "wiretransfer"
```

Comment:
Please, use the transaction **payment method name** to identify POI/PIA payments.

Reply:
Then the **if-else-statement** should also be obsolete as we only handle the POI/PIA-specific part in its own if.

Reply:
👍